website/integrations/services/owncloud/index.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Integrate with ownCloud
+sidebar_label: ownCloud
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is ownCloud
+
+> ownCloud is a free and open-source software project for content collaboration and sharing and syncing of files.
+>
+> -- https://owncloud.com
+
+:::note
+This guide focuses on ownCloud installations that are deployed using Docker. If you deployed ownCloud using a different mechanism, there might be some differences in the process.
+:::
+
+## Preparation
+
+The following placeholders are used in this guide:
+
+- `owncloud.company` is the FQDN of the ownCloud installation.
+- `authentik.company` is the FQDN of the authentik installation.
+
+:::info
+This guide focuses on setting up ownCloud using the [official OIDC plugin](https://github.com/owncloud/openidconnect). Advanced configuration details can be found in the [ownCloud admin manual](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html).
+:::
+
+## authentik configuration
+
+To support all ownCloud applications, multiple application/provider pairs are required—one each for the Web UI, Desktop application, Android application, and iOS application.
+
+The configuration for each application is nearly identical, except for the **Client ID**, **Client Secret**, and the **Redirect URI**, which are [predefined](https://doc.owncloud.com/server/latest/admin_manual/configuration/user/oidc/oidc.html#client-ids-secrets-and-redirect-uris) by ownCloud for the Desktop, Android, and iOS applications.
+
+### Create applications/providers
+
+Follow these steps to create the required application/provider pairs. You will need to repeat the process four times: once for the Desktop application, once for the Web UI, once for the Android application, and once for the iOS application.
+
+1. In the Admin interface, navigate to **Applications** > **Applications**.
+2. Use the new application wizard to create the application and provider, with the following settings:
+
+    - **Application**: Provide a descriptive name (e.g., `owncloud`, `owncloud-desktop`, `owncloud-android`, `owncloud-ios`), an optional group for the type of application, the policy engine mode, and optional UI settings.
+    - **Choose a Provider type**: Select **OAuth2/OpenID Provider** as the provider type.
+    - **Configure the Provider**: Provide a name (the default name will suffice for most users), the authorization flow to use for this provider, and the following required configurations. The settings for each of the four application/provider pairs are shown below:
+
+        - **Protocol settings:**
+          **Web UI:**
```

Comment:
```suggestion
          
          **Web UI:**
```

Reply:
The deploy preview shows the "Web UI" smashed up against "Protocol settings". A empty line between should fix it but please verify.